warplib/memory.cairo

```diff
@@ -51,6 +206,163 @@ func wm_write_felt{warp_memory: DictAccess*}(loc: felt, value: felt) -> (res: fe
+    let (fvalue) = u136_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_144{warp_memory: DictAccess*}(loc: felt, value: u144) -> (res: u144) {
+    let (fvalue) = u144_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_152{warp_memory: DictAccess*}(loc: felt, value: u152) -> (res: u152) {
+    let (fvalue) = u152_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_160{warp_memory: DictAccess*}(loc: felt, value: u160) -> (res: u160) {
+    let (fvalue) = u160_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_168{warp_memory: DictAccess*}(loc: felt, value: u168) -> (res: u168) {
+    let (fvalue) = u168_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_176{warp_memory: DictAccess*}(loc: felt, value: u176) -> (res: u176) {
+    let (fvalue) = u176_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_184{warp_memory: DictAccess*}(loc: felt, value: u184) -> (res: u184) {
+    let (fvalue) = u184_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_192{warp_memory: DictAccess*}(loc: felt, value: u192) -> (res: u192) {
+    let (fvalue) = u192_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_200{warp_memory: DictAccess*}(loc: felt, value: u200) -> (res: u200) {
+    let (fvalue) = u200_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_208{warp_memory: DictAccess*}(loc: felt, value: u208) -> (res: u208) {
+    let (fvalue) = u208_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_216{warp_memory: DictAccess*}(loc: felt, value: u216) -> (res: u216) {
+    let (fvalue) = u216_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_224{warp_memory: DictAccess*}(loc: felt, value: u224) -> (res: u224) {
+    let (fvalue) = u224_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_232{warp_memory: DictAccess*}(loc: felt, value: u232) -> (res: u232) {
+    let (fvalue) = u232_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_240{warp_memory: DictAccess*}(loc: felt, value: u240) -> (res: u240) {
+    let (fvalue) = u240_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
+func wm_write_248{warp_memory: DictAccess*}(loc: felt, value: u248) -> (res: u248) {
+    let (fvalue) = u248_to_felt(value);
+    dict_write{dict_ptr=warp_memory}(loc, fvalue);
+    return (value,);
+}
```

Comment:
Why do you add cairo 0 code?

Reply:
This code is just place holder in `memory.cairo` file, since all codes in that file is in cairo0,  I added older format of cairo codes. When we work on memory.cairo file then we'll replace it